README.unittests.rst

```diff
@@ -268,7 +268,7 @@ intended for production use!
 
     # configure the database
     sleep 20
-    docker exec -ti mysql mysql -u root -ppassword -w -e "CREATE DATABASE test_schema CHARSET utf8mb4; GRANT ALL ON test_schema.* TO scott;"
+    docker exec -ti mysql mysql -u root -p -w -e "CREATE DATABASE test_schema CHARSET utf8mb4; GRANT ALL ON test_schema.* TO scott;"
```

Comment:
this seems unrelated, should be reverted

Reply:
done